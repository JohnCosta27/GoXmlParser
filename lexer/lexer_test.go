package lexer_test

import (
	"testing"

	"github.com/gkampitakis/go-snaps/snaps"
	"johncosta.tech/xmlparse/lexer"
)

func TestBasicTokens(t *testing.T) {
  snaps.MatchSnapshot(t, lexer.Tokenize("<a>Tag</a>"))
}
