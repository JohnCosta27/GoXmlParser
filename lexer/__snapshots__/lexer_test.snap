
[TestBasicTokens - 1]
[]lexer.Token{
    {Token:"<", Token_content:""},
    {Token:"TEXT", Token_content:"a"},
    {Token:">", Token_content:""},
    {Token:"TEXT", Token_content:"Tag"},
    {Token:"</", Token_content:""},
    {Token:"TEXT", Token_content:"/a"},
    {Token:">", Token_content:""},
}
---
